README.md

```diff
@@ -1226,14 +1227,21 @@ An Azul build on Gitlab runs the `test`, `terraform`, `deploy`, `subscribe` and
 feature branches is `sandbox`, the protected branches use their respective
 deployments.
 
-## 9.1 The Sandbox Deployment
+## 9.1 CI/CD Systems
+
+- [SingleCell Development CI/CD Instance](https://gitlab.dev.singlecell.gi.ucsc.edu/)
```

Comment:
To be consistent, this URL should be https://gitlab.dev.singlecell.gi.ucsc.edu/ucsc/azul